### ISSUE TITLE
[Request] Support IconData.fontPackage optional parameter via CLI and fontify.yaml

### ISSUE THREAD

**Issue by jamieastley** [westracer/fontify#4]
Context: I currently have a generic Icon font package (lets call it 'MyCustomFonts') that I intend add as a dependency and import across multiple projects for work. 

Currently if I were to use fontify to generate the Icon class, I would have to include the generated *.otf file in the apps' `pubspec.yaml` for the icons to correctly render, but this isn't ideal when I just want to be able to import my custom font package like any other dependency.

Alternatively, if the `pubspec.yaml` of `MyCustomFonts` includes the font file asset, and then in the generated Icon class we set the `fontPackage` parameter for each `IconData` to point to it's own package, eg:
```dart
static const _kFontFamily = 'MyFontFamily';

static const _kFontPackage = 'MyCustomFonts';

static const IconData some_icon =
      IconData(0xe000, fontFamily: _kFontFamily, fontPackage: _kFontPackage);
```

it means not having to manually include the *.otf in the new apps we want to use the font package with, and the only thing to do is just add the `MyCustomFonts` package to the new apps `pubspec.yaml` and the icons are good to go.

I've currently forked fontify to hardcode the value I want for this variable in `flutter_class_gen.dart` and considered making a PR, but I don't quite understand the rest of the package to fully implement setting this via a CLI argument and `fontify.yaml` configuration.

**Comment by westracer** [comment]
Hello, @jamie1192. This is my bad, I totally ignored the "fontPackage" property. I'll implement it asap in the next release. Thank you.

**Comment by westracer** [comment]
Implemented and released in v0.1.0

**Comment by jamieastley** [comment]
Awesome, thank you so much for the very quick turn around!
